internal/backend/backend.go

```diff
@@ -122,6 +124,31 @@ func (b *Backend) RemoveUser(ctx context.Context, userID string, removeFiles boo
 	return nil
 }
 
+func (b *Backend) GetCounts(ctx context.Context, userID string) (map[imap.MailboxID]int, error) {
```

Comment:
Can we get a bit more descriptive name here?

Reply:
Good point -- "Counts" means a specific thing on the proton API, but here in gluon it's unclear. Changed to `GetMailboxMessageCounts`.